networking/networking_operators/aws_load_balancer_operator/configuring-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,21 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="configuring-aws-load-balancer-operator"]
+= Configuring the AWS Load Balancer Operator
+include::_attributes/common-attributes.adoc[]
+:context: aws-load-balancer-operator
+
+toc::[]
+
+include::modules/configuring-egress-proxy.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_{context}"]
+== Additional resources
```

Comment:
This should just be a list that appears after the included module (we save the `==` style for the end of a full assembly), but, not blocking merge for this:
```suggestion
[role="_additional-resources"]
.Additional resources
```